### ISSUE TITLE
Wrong translation context for "State"

### ISSUE THREAD

**Issue by yostashiro** [odoo/odoo#128219]
### Impacted versions

16.0 and others

### Current situation

```
#. module: base
#: model:ir.model.fields,field_description:base.field_base_language_export__state
#: model:ir.model.fields,field_description:base.field_base_partner_merge_automatic_wizard__state
#: model:ir.model.fields,field_description:base.field_res_partner__state_id
#: model:ir.model.fields,field_description:base.field_res_users__state_id
#: model:ir.model.fields,field_description:base.field_res_users_deletion__state
#: model_terms:ir.ui.view,arch_db:base.res_partner_view_form_private
#: model_terms:ir.ui.view,arch_db:base.view_company_form
#: model_terms:ir.ui.view,arch_db:base.view_country_state_form
#: model_terms:ir.ui.view,arch_db:base.view_country_state_tree
#: model_terms:ir.ui.view,arch_db:base.view_module_filter
#: model_terms:ir.ui.view,arch_db:base.view_partner_address_form
#: model_terms:ir.ui.view,arch_db:base.view_partner_form
#: model_terms:ir.ui.view,arch_db:base.view_res_bank_form
msgid "State"
msgstr ""
```
https://github.com/odoo/odoo/blob/2b7e86da909187a002939cc29b0e5a76b95b9136/odoo/addons/base/i18n/base.pot#L20613-L20628

`#: model_terms:ir.ui.view,arch_db:base.view_module_filter` is about situation/status, while the others are about the state as in `res.country.state`, and it forces us to assign the same translation.

Currently the Japanese translation of "State" in apps search view is wrong.

![image](https://github.com/odoo/odoo/assets/7766939/95fe7aec-7a66-4abd-88c3-0febb60ea79e)

### Expected situation

Separate the entries like below (wasn't sure it's possible):

```
#. module: base
#: model:ir.model.fields,field_description:base.field_base_language_export__state
#: model:ir.model.fields,field_description:base.field_base_partner_merge_automatic_wizard__state
#: model:ir.model.fields,field_description:base.field_res_partner__state_id
#: model:ir.model.fields,field_description:base.field_res_users__state_id
#: model:ir.model.fields,field_description:base.field_res_users_deletion__state
#: model_terms:ir.ui.view,arch_db:base.res_partner_view_form_private
#: model_terms:ir.ui.view,arch_db:base.view_company_form
#: model_terms:ir.ui.view,arch_db:base.view_country_state_form
#: model_terms:ir.ui.view,arch_db:base.view_country_state_tree
#: model_terms:ir.ui.view,arch_db:base.view_partner_address_form
#: model_terms:ir.ui.view,arch_db:base.view_partner_form
#: model_terms:ir.ui.view,arch_db:base.view_res_bank_form
msgid "State"
msgstr ""

#. module: base
#: model_terms:ir.ui.view,arch_db:base.view_module_filter
msgid "State"
msgstr ""
```

Or, change the original string to 'Status' or something.

@qrtl

**Comment by niyasraphy** [comment]
https://github.com/odoo/odoo/issues/125949

**Comment by mart-e** [comment]
Hello,

You are right, this is a frequent issue when using terms with several meaning.
The problem is that your suggestion is **not** possible, it would make the .pot file invalid and being rejected by Transifex.

In this specific case, the problem was the opposit, that the country state was labeld "Status" while it should have been "State".

I have fixed it last week at  #130357 so it should be correct on new databases or if you upgrade your base module (backup before you do :crossed_fingers: ).
Let me know if not solved.
